docs/scout.txt

```diff
@@ -0,0 +1,198 @@
+.. _laravel-scout:
+
+============================
+Integrated Search with Scout
+============================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: php framework, odm, code example, text search, atlas
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use the Laravel Scout feature in
+your {+odm-long+} application. Scout allows you to implement full-text
+search on your Eloquent models. To learn more, see `Laravel Scout
+<https://laravel.com/docs/{+laravel-docs-version+}/scout>`__ in the
+Laravel documentation.
+
+The Scout Integration for {+odm-long+} provides the following
+functionality:
+
+- Provides an abstraction to create search indexes on documents in
+  MongoDB collections and other search engines. In MongoDB, this feature
+  allows you to create :atlas:`Atlas Search indexes
+  </atlas-search/manage-indexes/>`.
+
+- Allows you to automatically replicate data from MongoDB into a
+  search engine such as `Meilisearch <https://www.meilisearch.com/>`__
+  or `Algolia <https://www.algolia.com/>`__. You can use a MongoDB Eloquent
+  model as the source to import and index.
+
+.. note:: Deployment Compatibility
+
+   You can use Laravel Scout only when you connect to MongoDB Atlas
+   clusters. This feature is not available for self-managed or
+   serverless deployments.
+
```

Comment:
S: since this page reads like a step-by-step tutorial, you could make it a tutorial (with `.. procedure:`) or just include a numbered list here with each of the steps (i.e., "Complete the following steps to configure Scout and create a searchable collection:
1. :ref:`Install Scout Package <link to section>`
2. :ref: `Add the Searchable Trait to Your Model <link>`
3. ..."

Reply:
I'll make it a procedure